student-library/src/main/java/tds/student/sql/data/OpportunityItem.java

```diff
@@ -58,6 +53,7 @@ public long getItemKey () {
     return _itemKey;
   }
 
+  @JsonProperty("ItemKey")
```

Comment:
@malaffoon In general things that you see in the TDS projects that were created prior to August 2016 have style guidelines that do not follow what we've seen in other Java projects.  Due to current timelines we chose not to go through and "fix" them to follow more standard practices in Java.

And even though this repository was created post August 2016 it is actually a library that used to be embedded in TDS_Student.